### DISCUSSION TITLE
[우제민] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by woojmn** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#29] [2024년도 과제 제출]
![image](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/assets/95162423/e5c1245a-39bf-4b98-9027-f86b887f938a)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다. 

아래 링크에서 설문조사를 완료해 주시길 바랍니다.
https://forms.gle/DDwJCEf9Zbs3fxFi8

감사합니다.
